SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -247,16 +285,29 @@ bool USpatialLatencyTracer::BeginLatencyTrace_Internal(const AActor* Actor, cons
 
 	WriteKeyFrameToTrace(&NewTrace, FString::Printf(TEXT("Begin trace : %s"), *FunctionName));
 
+	// For non-spatial tracing
+	const improbable::trace::SpanContext& TraceContext = NewTrace.context();
```

Comment:
Nit: might be cute to add a little private function or anonymous namespace function that takes (improbable::trace::SpanContext&, FSpatialLatencyPayload&) and does this:

OutLatencyPayload.TraceId = TArray<uint8_t>((const uint8_t*)&TraceContext.trace_id()[0], sizeof(improbable::trace::TraceId));
OutLatencyPayload.SpanId = TArray<uint8_t>((const uint8_t*)&TraceContext.span_id()[0], sizeof(improbable::trace::SpanId));

Not really important though.